bzt/utils.py

```diff
@@ -627,6 +632,30 @@ def install(self):
             except BaseException as exc:
                 raise exc
 
+    def install_with_mirrors(self, dest, suffix):
+        self.log.info("Will try to install %s into %s", self.tool_name, dest)
+        downloader = request.FancyURLopener()
+        tool_dist = tempfile.NamedTemporaryFile(suffix=suffix, delete=False)  # delete=False because of Windows
+        mirrors = self.mirror_manager.mirrors()
+        while True:
```

Comment:
So it's not for avoiding infinity loops, but only for better readability. Ok, got it. Readability counts.